static/app/components/organizations/pageFilters/container.tsx

```diff
@@ -122,15 +130,13 @@ function Container({
   //
   // This happens when we mount the container.
   useEffect(() => {
-    // We can initialize before ProjectsStore is fully loaded if we don't need to
-    // enforce single project.
-    if (!projectsLoaded && (shouldForceProject || enforceSingleProject)) {
+    if (!projectsLoaded) {
       return;
     }
 
     doInitialization();
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [projectsLoaded, shouldForceProject, enforceSingleProject]);
+  }, [projectsLoaded]);
```

Comment:
In order for this fix to work correctly, we need to initialize page filters _after_ projects have been loaded. In my testing, this hasn't been an issue. There's minimal user impact since currently page filters are disabled until projects are loaded.